ios/ios_app_delegate.mm

```diff
@@ -0,0 +1 @@
+
```

Comment:
We need an empty file?

Reply:
It was a test to overcome a message during runtime (there was and still is a warning during runtime that you need to set a property in the app delegate for ui view or similar. Not exactly sure how I can solve this therefore I've just created this file since we'll probably need it.)

Reply:
Ok, so let's keep it for now.